vispy/testing/_runners.py

```diff
@@ -292,7 +310,8 @@ def test(label='full', extra_arg_string=''):
     elif label in backend_names:
         runs.append([partial(_unit, label, extra_arg_string), label])
     if label in ('full', 'examples'):
-        runs.append([_examples, 'examples'])
+        runs.append([partial(_examples, extra_arg_string),
+                    'examples'])
     if label in ('full', 'unit', 'nobackend'):
         runs.append([partial(_unit, 'nobackend', extra_arg_string),
```

Comment:
because in that case, it should be interpreted as being passed to `_unit` not to `_examples`